.github/workflows/stackhpc-container-image-build.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           repository: stackhpc/kayobe
-          ref: refs/heads/stackhpc/wallaby
```

Comment:
Revert me

Reply:
Didn't need this anyway if you specify a regex, but having the kolla_enable_caso flag makes sure the image is built as the standard set.